astro.config.mjs

```diff
@@ -9,14 +9,16 @@ import { loadEnv } from "vite";
 import liciousI18n from "@astrolicious/i18n";
 const {
   SANITY_API_READ_TOKEN,
+  ASTRO_SITE_URL,
   SANITY_STUDIO_DATASET,
-  SANITY_STUDIO_PROJECT_ID
+  SANITY_STUDIO_PROJECT_ID,
+  SANITY_VISUAL_EDITING_ENABLED
```

Comment:
We should document these environment variables and what they do. Having them in .env.example is enough